app/styles/ui/_pull-request-diff.scss

```diff
@@ -0,0 +1,4 @@
+.pull-request-diff-viewer {
+  height: 500px;
```

Comment:
This static height is likely temporary and just to make the diff not expand past the dialog during development of this component. I will have to eventually determine the logic for dynamically setting this height based on the users window size so that the dialog can use as much room as is available within whatever min/max constraints we figure out for the diff (similarly for width).